### PR TITLE
feat: 🎸 add inline alignment support to checkbox and radio

### PR DESCRIPTION
**Pull request by DhariniJeeva**
This pr adds inline alignment support to radio and checkbox comp

Screen Shot 2021-07-20 at 3 53 21 PM

### CODE REVIEW

**Review by gsusmi — COMMENTED** [review]
Storybook associated with these components will need to be updated to document this new attribute. Check out `yarn storybook` command in the addon.

**Comment by DhariniJeeva** [comment]
first time updating in the storybook. Please let me know if I missed anything in the storybook components.

**Review by randallmorey — CHANGES_REQUESTED** [review]
Inline is unique enough that it may merit its own story section for each of these components.

File: addons/rose/addon/styles/rose/components/form/_radio.scss [hunk 1]
```diff
@@ -21,6 +21,12 @@ $radio-svg: (
 );
 
 $data-svg-prefix: 'data:image/svg+xml;utf-8,';
+.radio-group-inline {
```
Comment (gsusmi) [inline addons/rose/addon/styles/rose/components/form/_radio.scss:24]: I couldn't find where this style is being used. Can you point me to it?

Comment (DhariniJeeva) [inline addons/rose/addon/styles/rose/components/form/_radio.scss:24]: my bad, this is incorrect. This should be `inline` which comes from `addons/rose/addon/components/rose/form/radio/radio/index.hbs` line 2

File: addons/rose/addon/styles/rose/components/form/_radio.scss [hunk 2]
```diff
@@ -21,8 +21,13 @@ $radio-svg: (
 );
 
 $data-svg-prefix: 'data:image/svg+xml;utf-8,';
-
-.rose-form-radio {
+  .rose-form-radio {
```
Comment (randallmorey) [inline addons/rose/addon/styles/rose/components/form/_radio.scss:24]: Did indentation change here?

File: addons/rose/addon/components/rose/form/checkbox/index.hbs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div class="rose-form-checkbox{{if @error " error"}}">
+<div class="rose-form-checkbox{{if @error " error"}} {{if @inline "inline"}}">
```
Comment (randallmorey) [inline addons/rose/addon/components/rose/form/checkbox/index.hbs:1]: I'd follow the pattern with the error class and include the space in the conditional string just to keep the markup tidy.

**Review by randallmorey — CHANGES_REQUESTED** [review]
Couple of minor points.  Also was wondering if you could run the formatter `yarn format` on Rose to ensure everything is consistent.

Otherwise great work!

**Review by randallmorey — APPROVED** [review]

File: addons/rose/addon/components/rose/form/checkbox/index.hbs (hunk 3 above)
Comment (DhariniJeeva) [reply addons/rose/addon/components/rose/form/checkbox/index.hbs:1]: sure, I will fix this and the formatting

**Review by calcaide — APPROVED** [review]
LGTM! Great work!

**Pull request merged**